src/scripts/main.js

```diff
@@ -1,3 +1,359 @@
 'use strict';
 
-// write your code here
+let board = [];
+const boardSize = 4;
+let score = 0;
+let gameOver = false;
+let gameStarted = false;
+
+const startButton = document.querySelector('.start');
+const restartButton = document.querySelector('.restart');
+const scoreEl = document.querySelector('.game-score');
+
+const startMessage = document.querySelector('.message.message-start');
+const winMessage = document.querySelector('.message-win');
+const loseMessage = document.querySelector('.message-lose');
+
+const gameBoard = document.querySelector('.game-field');
+const rows = gameBoard.querySelectorAll('.field-row');
+
+window.addEventListener('load', () => {
+  initializedBoard();
+  generateTwoTile();
+  upDateBoard();
+});
+
+document.addEventListener('keydown', changeKey);
+
+startButton.addEventListener('click', () => {
+  startGame();
+}
+);
+
+restartButton.addEventListener('click', () => {
+  restartGame();
+});
+
+function startGame() {
+  toggleButton(startButton, restartButton);
+  hideElem(startMessage);
+  resetGame();
+  generateTwoTile();
+  upDateBoard();
+}
+
+function restartGame() {
+  toggleButton(restartButton, startButton);
+  startGame();
+  hideElem(loseMessage);
+}
+
+function resetGame() {
+  board = [];
+  initializedBoard();
+  score = 0;
+  gameOver = false;
+}
+
+function changeKey(evt) {
+  evt.preventDefault();
+
+  if (!gameStarted) {
+    startGame();
+  }
+
+  gameStarted = true;
+
+  if (!gameOver) {
+    switch (evt.key) {
+      case 'ArrowUp':
+        moveUp();
+        break;
+      case 'ArrowDown':
+        moveDown();
+        break;
+      case 'ArrowLeft':
+        moveLeft();
+        break;
+      case 'ArrowRight':
+        moveRight();
+        break;
+    }
+    generateTwoTile();
```

Comment:
only one cell should appear when user press some button

Reply:
The game start with one cell?